selene_sdk/update_seqweaver.py

```diff
@@ -0,0 +1,152 @@
+"""
```

Comment:
this is nicely written! I think you can put this in `selene_sdk/scripts` for the time being, but at some point I'd expect you would move this into a totally separate repository in `FunctionLab`, complete with downloadable files and a command-line script to be able to run this code and reproduce your work! 

it's a good reminder that maybe we should start compiling a list of projects that use selene for future users to be able to refer back to when they start their own projects